### PR TITLE
Update eip-725.md

### PR DESCRIPTION
**Pull request by YamenMerhi**
## Description

As i mentioned in the issue, @frozeman , @CJ42 and I found that the better way to setData in ERC725Y is by passing an array of data which can be set in one transaction, that way we can skip the need to to make a contract call each time we want to set singular form of data. Same applies for getData.

## Issue

For more information check the issue #3857

### CODE REVIEW

**Review by frozeman — APPROVED** [review]

File: EIPS/eip-725.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 eip: 725
-title: Smart Contract Based Account
+title: ERC-725 Smart Contract Based Account
```
[the remark this answers is not in the document]
Comment (frozeman) [reply EIPS/eip-725.md:3]: I would even rename it to `General key-value store and execution standard`

**Comment by poojaranjan** [comment]
It appears to me that this pull request is merged by the bot because it is approved by one of the authors. But, this merge reverts the changes done in the "title" earlier with #3718. It is also highlighted by @axic in the earlier comment, so merge shouldn't be done by the bot. 

As per the above comments, I suspect, the author would have not wanted this merge to happen. Can/should it be reverted?

@frozeman @MicahZoltu @lightclient @axic

**Comment by MicahZoltu** [comment]
Someone can just submit another PR that fixes the title again.  Looks like it was just an unfortunate mistake/accident but easily rectified.

**Comment by poojaranjan** [comment]
Agreed, this sounds simpler :)

**Pull request merged**